opencensus/stats/view_data.py

```diff
@@ -72,13 +73,17 @@ def end(self):
 
     def get_tag_map(self, context):
         """function to return the tag map based on the context"""
-        if context.items() <= self.tag_map.items():
-            return self.tag_map
-        else:
-            tags = self.tag_map
-            for tag_key, tag_value in context.items():
-                tags[tag_key] = tag_value
-            return tags
+        if self.tag_map is not None:
+            if context.items() <= self.tag_map.items():
+                return self.tag_map
+            else:
+                tags = self.tag_map
+                for tag_key, tag_value in context.items():
+                    tags[tag_key] = tag_value
+                return tags
+        else:  # pragma: NO COVER
+            logging.warning("Tag Map cannot be none",
+                            self.tag_map)
```

Comment:
Just a message in logging is enough, it will throw `TypeError` when `self.tag_map` is None in the logging call here.